app/models/user.rb

```diff
@@ -1,8 +1,15 @@
 # frozen_string_literal: true
 
 class User < ApplicationRecord
-  # Include default devise modules. Others available are:
-  # :confirmable, :lockable, :timeoutable, :trackable and :omniauthable
   devise :database_authenticatable, :registerable,
          :recoverable, :rememberable, :validatable
+  devise :omniauthable, omniauth_providers: %i[github]
```

Comment:
diveseで元から記述してあるシンボルと、OmniAuthで追記したシンボルを分けて書いていますが、下のように合わせて記述するのと、どちらが一般的でしょうか?
```suggestion
  devise :database_authenticatable, :registerable,
         :recoverable, :rememberable, :validatable
         :omniauthable, omniauth_providers: %i[github]
```

Reply:
個人的経験では後者ですが、前者でも動いてるならいいんじゃないかなって思います。可読性は前者の方が高そうですし。

Reply:
了解です!